bigframes/ml/llm.py

```diff
@@ -289,12 +295,16 @@ def __init__(
         self.model_name = model_name
         self.version = version
         self.session = session or bpd.get_global_session()
-        self._bq_connection_manager = clients.BqConnectionManager(
-            self.session.bqconnectionclient, self.session.resourcemanagerclient
+        self._bq_connection_manager = (
+            None
+            if self.session._skip_bq_connection_check
+            else clients.BqConnectionManager(
+                self.session.bqconnectionclient, self.session.resourcemanagerclient
+            )
         )
 
         connection_name = connection_name or self.session._bq_connection
-        self.connection_name = self._bq_connection_manager.resolve_full_connection_name(
+        self.connection_name = clients.BqConnectionManager.resolve_full_connection_name(
```

Comment:
classmethod should be limited. Just a module method is enough.

go/pystyle#decorators-decision

Reply:
it was an existing classmethod, moved outside to the module level.